Gemfile

```diff
@@ -40,6 +40,7 @@ group :development, :test do
   gem 'capybara', '~> 2.13'
   gem 'selenium-webdriver'
   gem 'rubocop', '0.58.2'
+  gem 'rspec'
```

Comment:
Me rompia rspec entonces agregué la gema

Reply:
:+1: but you need rspec only on your `test` environment.